src/components/PageNavLink/PageNavLink.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+type Props = {
+  to: string,
+  text: string,
+};
+
+export const PageNavLink: React.FC<Props> = ({ to, text }) => {
+  return (
+    <NavLink
+      to={to}
+      className={({ isActive }) => classNames(
+        'navbar-item', { 'is-active': isActive },
+      )}
```

Comment:
you have class is-active in `li` element, here you can use `<Link>` and remove `is-active` class